packages/ListBox/test/cypress/ListBox.a11y.cypress.js

```diff
@@ -0,0 +1,86 @@
+import { getStoryUrlPrefix } from "../../../../.storybook/storyTree";
+import { toggleDropdown } from "../helpers/toggleHelpers";
+import selectors from "../helpers/selectors";
+
+const storyPrefix = `${getStoryUrlPrefix("ListBox")}`;
+
+function arrowUpKeyPress() {
+  cy.focused().type("{uparrow}", { force: true });
+}
+
+function arrowDownKeyPress() {
+  cy.focused().type("{downarrow}", { force: true });
+}
+
+function optionHasFocus(optionName) {
+  cy.get(selectors.popoverList)
+    .findByText(optionName)
+    .should("have.focus");
+}
+
+describe("ListBox MultiSelect a11y", () => {
+  beforeEach(() => {
+    cy.visitStorybook(`${storyPrefix}-backyard-tests--a-11-y-tests-story`);
+    toggleDropdown();
+    cy.wait(350);
```

Comment:
Wait's needed throughout this test to help with focus timeouts and popover animation.

This also prevents a lot of flakiness with keyboard testing

Reply:
😦  But `toggleDropdown()` already waits until the popover is visible...?
https://github.com/acl-services/paprika/blob/master/packages/ListBox/test/helpers/toggleHelpers.js#L9